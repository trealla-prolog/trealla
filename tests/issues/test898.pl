:- initialization(main).

main :-
	catch(call((true->false,1)), E, true), !, write(E), nl.
main :-
	write(nok), nl.
