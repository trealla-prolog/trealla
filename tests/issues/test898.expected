error(type_error(callable,(true->false,1)),call/1)
